docker-compose-ci.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3.3"
+services:
+  postgres:
+    image: postgres:9.4
+    logging:
+      driver: none
+    environment: 
+      POSTGRES_HOST_AUTH_METHOD: trust
+    volumes: 
+      - ./psql-users.sh:/docker-entrypoint-initdb.d/psql-users.sh
+
+  app:
+    image: quay.io/ncigdc/jenkins-agent:multipython
+    environment:
+      SSH_AUTH_SOCK: $SSH_AUTH_SOCK
+      HTTPS_PROXY: http://cloud-proxy:3128
+      HTTP_PROXY: http://cloud-proxy:3128
+    volumes:
+    - .:/home/jenkins
+    - $SSH_AUTH_SOCK:$SSH_AUTH_SOCK
+    command: bash -c "./wait-for-it.sh localhost:5432 -t 120  &&  pip install tox -q --user && tox --recreate"
```

Comment:
Random idea, but I wonder if installing tox before running wait-for-it might allow Postgres to start up in parallel with the tox install and make this go a little faster.

Actually, we might just want to build tox into the Jenkins agent image. Maybe you already did that...

Reply:
I will build tox into the image, that does make sense and I remember I was going to do it but forgot. Might make the tests a tiny bit faster